diff --git a/flow/scripts/report_metrics.tcl b/flow/scripts/report_metrics.tcl
index b885d9f3..b051112f 100644
--- a/flow/scripts/report_metrics.tcl
+++ b/flow/scripts/report_metrics.tcl
@@ -21,8 +21,8 @@ proc report_metrics { when {include_erc true} {include_clock_skew true} } {
     puts "$when report_clock_skew"
     puts "--------------------------------------------------------------------------"
     report_clock_skew
-    report_clock_skew_metric
-    report_clock_skew_metric -hold
+    #report_clock_skew_metric
+    #report_clock_skew_metric -hold
   }
   
   puts "\n=========================================================================="
